ndk-examples/examples/jni_audio.rs

```diff
@@ -18,7 +17,7 @@ fn enumerate_audio_devices() -> Result<(), Box<dyn std::error::Error>> {
 
     let audio_manager = env
         .call_method(
-            native_activity.activity(),
+            ctx.context() as jni::sys::jobject,
```

Comment:
```suggestion
            ctx.context().cast(),
```

Should work here too?